app/views/triple/index.html.erb

```diff
@@ -30,6 +30,7 @@
 <div class="add-button-container">
     <button id="add-button" onclick="addEntry()">Create Triple</button>
     <button id="export-csv">Download CSV</button>
+    <button onclick="window.location.href='http://127.0.0.1:3042/api/v1/graph'">View JSON</button>
```

Comment:
this code will work only in a local environment. 
If we host the app online then 127.0.0.1 won't be the root of the app anymore